spec/models/manageiq/providers/azure/cloud_manager/provision_spec.rb

```diff
@@ -80,8 +83,20 @@
       end
 
       context "nic settings" do
-        it "with nic" do
-          subject.options[:vm_target_name] = name
+        it "with floating_ip" do
```

Comment:
I think it would be great to describe what this test actually expects, like
`it "with floating_ip provided assigns floating_ip to clone_task`  - not sure if this is correct :smile: 

right now the `it`s here are more like a context